src/styles/info-label.scss

```diff
@@ -81,19 +85,20 @@ $color-accents: (
   }
 
   &--display {
-    height: $fd-info-label-display-height;
+    border-color: var(--fdInfo_Label_Display_Border_Color);
+    background-color: var(--fdInfo_Label_Display_Background_Color);
+    color: var(--fdInfo_Label_Display_Text_Color);
     min-width: $fd-info-label-display-height;
```

Comment:
same as the other comment, I can't find anything about min-width

Reply:
removed